From: xionglinlin <xionglinlin@uniontech.com>
Date: Sun, 29 Sep 2024 11:44:52 +0800
Subject: fix: remove the restart icon

remove the restart icon from the right-click menu

Log: remove the restart icon
Bug: https://pms.uniontech.com/bug-view-270971.html
---
 src/modules/notificationitem/dbusmenu.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/modules/notificationitem/dbusmenu.cpp b/src/modules/notificationitem/dbusmenu.cpp
index 10e33c5..e29409f 100644
--- a/src/modules/notificationitem/dbusmenu.cpp
+++ b/src/modules/notificationitem/dbusmenu.cpp
@@ -259,8 +259,6 @@ void DBusMenu::fillLayoutProperties(
         case BII_Restart:
             appendProperty(properties, propertyNames, "label",
                            dbus::Variant(_("Restart")));
-            appendProperty(properties, propertyNames, "icon-name",
-                           dbus::Variant("view-refresh"));
             break;
         case BII_Exit:
             appendProperty(properties, propertyNames, "label",
